lang/de_DE.json

```diff
@@ -20,7 +20,7 @@
         "On": "Ein",
         "Off": "Aus",
         "Ok": "OK",
-        "New Timer": "Neuer Kurzzeitwecker",
```

Comment:
While "Kurzzeitwecker" seems to be an
accurate translation I never heard this word
before. Everyone here understands "Timer".